sw/device/silicon_creator/rom/doc/rom_overview_specification.md

```diff
@@ -0,0 +1,270 @@
+# Mask ROM Specification
+
+
+## Overview
+
+This document is a high-level overview of the features and responsibilities of the Mask ROM.
+
+This document is not a comprehensive specification.
+When appropriate, it refers to more complete specifications for various mask ROM responsibilities.
+These specifications include:
+
+
+
+*   [Signature Verification Module [sigverify]](#sigverify)
+*   [Boot Policy Module [bootpolicy]](#bootpolicy)
+*   [Address space unlock module [unlock]](#unlock)
+*   [Bootstrap module [bootstrap]](#bootstrap)
+*   [Bootstrap decoding](#strapping)
+*   [Shutdown Module [shutdown]](#shutdown)
+*   [Chip-Specific Startup [startup]](#startup)
+*   [Mask ROM Pin Configuration [pinconfig]](#pinconfig)
```

Comment:
My preference would be to have empty links for pages or sections that don't exist yet. It took me a minute to work out these links weren't just broken but intended to point to future content.

I'd link to have a link checker in CI at some point that is fragment aware. If I were to set this up, empty links would not cause a failure but broken fragments would.

```suggestion
*   [Signature Verification Module [sigverify]]()
*   [Boot Policy Module [bootpolicy]]()
*   [Address space unlock module [unlock]]()
*   [Bootstrap module [bootstrap]]()
*   [Bootstrap decoding]()
*   [Shutdown Module [shutdown]]()
*   [Chip-Specific Startup [startup]]()
*   [Mask ROM Pin Configuration [pinconfig]]()
```

Reply:
Done.